mealpy/mealpy.py

```diff
@@ -40,7 +40,8 @@ def load_config():
         'email_address': strictyaml.Email(),
         'use_keyring': strictyaml.Bool(),
     })
-    root_dir = path.abspath(path.dirname(__file__))
+    # TODO: Revert '../' path (temp bugfix)
+    root_dir = path.abspath(path.dirname(__file__) + '../')
```

Comment:
This didn't work for me, probably needs a path.join instead of string formatting.
`/home/ipwnponies/repos/mealpy/mealpy../config.template.yaml`

We can ignore this bug in the stopgap, if #27 can be merged shortly.

Reply:
Hmm, it does work for me though. However #27 does not work for me, it seems that `ROOT_DIR` there isn't correct.

Reply:
From the [docs](https://docs.python.org/3/library/os.path.html?highlight=path#os.path.split)
> Trailing slashes are stripped from head unless it is the root (one or more slashes only).

It seems to be working by coincidence for you. Path manipulation is tricky, which is why `os.path.join` should be used instead of string manipulation.